internal/maintenance/job_cleaner_test.go

```diff
@@ -71,7 +72,7 @@ func TestJobCleaner(t *testing.T) {
 		cleaner.Logger = riverinternaltest.LoggerWarn(t) // loop started/stop log is very noisy; suppress
 		cleaner.TestSignals = JobCleanerTestSignals{}    // deinit so channels don't fill
 
-		runStartStopStress(ctx, t, cleaner)
```

Comment:
Just moved this into a test package so we can use it across multiple packages to test start/stop services.